src/testUtils/TestRig.react.js

```diff
@@ -57,10 +57,17 @@ export default class TestRig {
     var element = this.domNode.find(selector)
     this.fillInElement(element, value)
   }
-  toggleCheckbox(selector) {
-    var element = this.domNode.find(selector) // Selector should be specific to the checkbox
-    $(element).prop('checked', !($(element)[0].checked))
-    TestUtils.Simulate.change(element[0])
+  toggleCheckbox(element) {
+    element.checked = !(element.checked)
+    TestUtils.Simulate.change(element)
+  }
+  toggleRadioButton(element) {
+    element.checked = !(element.checked)
+    TestUtils.Simulate.change(element)
```

Comment:
@andela-tkomolafe Two things:
1.  I think it's cumbersome to pass in the `element` every time. It means that using this requires the dev to drill down from the selector manually in every test they write; and,
2. These two lines of code are identical to the two lines in `toggleCheckbox`.

Take a look at how I handled this in my old PR: you pass in the `selector`, which is a lot simpler than the `element`, and the method is generalized to `toggleElement`. Then `toggleCheckbox` and `toggleRadioButton` can be defined in the `constructor` as aliases of `toggleElement`.